tket/README.md

```diff
@@ -87,8 +87,8 @@ If using gcc and with gcovr installed (`pip install gcovr`), a test coverage
 report can be generated using the following sequence of commands:
 
 ```shell
-conan build tket --user=tket --channel=stable -s build_type=Debug --build=missing -o "boost/*":header_only=True -o "tket/*":profile_coverage=True -o "test-tket/*":with_coverage=True -o with_test=True -of build/tket
+conan build tket --user=tket --channel=stable -s build_type=Debug --build=missing -o boost/*:header_only=True -o tket/*:profile_coverage=True -o test-tket/*:with_coverage=True -o with_test=True -of build/tket
```

Comment:
See above (for quotes around arguments containing asterisks).

Reply:
Done.